src/foraging_gui/Foraging.py

```diff
@@ -3917,8 +3918,8 @@ def _Start(self):
                     self.Metadata_dialog.ProjectName.setCurrentIndex(index)
                     self.Metadata_dialog._show_project_info()
                     logging.info('Setting Project name: {}'.format(project_name))
-                    add_default = False
-            if add_default:
+                    self.add_default_project_name = False
+            if self.add_default_project_name:
```

Comment:
This logic seems wrong. We only want to add the default project name if `self.add_default_project_name` is True, AND the (project_name is not None).

So I would add back `add_default`, and then change the if block to `if self.add_default_project_name and add_default`